driver/pace/driver/initialization.py

```diff
@@ -125,26 +122,18 @@ class RestartConfig(Initializer):
     Configuration for restart initialization.
     """
 
-    path: str
-
-    @property
-    def start_time(self) -> datetime:
-        return datetime(2000, 1, 1)
+    path: str = "."
+    start_time: datetime = datetime(2000, 1, 1)
```

Comment:
We can ignore this for now, but we should probably attach the model time to DriverState in the future and save it in the netcdfs.